### DISCUSSION TITLE
Question... How do i localize the backoffice from custom language V14

### DISCUSSION THREAD

**Discussion by YoussefDarwish0** [umbraco/Umbraco-CMS#16787] [Features and ideas]
I want to localize the backoffice to arabic. I want to understand how to add my own language! 
From the users section you can change to built in languages like hebrew etc... Where do these files exist.
Any clue on how i could do this?

**Comment by nul800sebastiaan** [comment]
FYI: the discussions board here on GitHub is only meant to create new feature requests for Umbraco CMS. 

This sounds like an excellent question for the forums or Discord where our friendly community can help you find the best solution for your requirements. 

Make sure to head on over to https://our.umbraco.com or https://discord.umbraco.com and ask follow up questions there.
